packages/marko-web-theme-monorail/components/nodes/content.marko

```diff
@@ -33,10 +36,23 @@ $ if (withSection && alias && withPrimarySectionModifier) {
   modifiers.push(`section-${primarySection.alias}`);
 }
 
-$ const imageLink = primaryImage.src ? { href: content.siteContext.path, attrs: linkAttrs } : false;
-
 $ const withSponsored = primarySection && primarySection.name === 'Sponsored' && input.withSponsored;
 
+// linkOffsite is an object like the following:
+/**
+  {
+    webinar: 'linkUrl',
+    event: 'website'
+  }
+  Keys are content types (all lowercase), values are the field to use when available on that type.
+**/
+$ const linkOffsite = getAsObject(site, "config.linkOffsite");
+$ const linkField = get(linkOffsite, contentType);
+$ const externalLink = get(content, linkField) || get(content, "externalContentUrl") || null;
```

Comment:
@Shinsina does this need to be config driven or can we just use `get(content, "externalContentUrl") || null(`?  It would only be present on the webinar & event types based on the fragment below.